### PR TITLE
Optimization

### PR DESCRIPTION
**Pull request by SFFan123**
2nd time...

### CODE REVIEW

File: src/app/emotes.json [hunk 1]
```diff
@@ -1,312 +1,312 @@
 {
-  "Bosch": {
```
Comment (zaanposni) [inline src/app/emotes.json]: Please stop this weird format REEE

Comment (SFFan123) [reply src/app/emotes.json]: why you complaining about the format of a line that gets removed?
do you want that format to carry over to the new version
or
do you dslike the format that gets removed

Comment (zaanposni) [reply src/app/emotes.json]: I dislike the format of the file overall
[https://github.com/FI18-Trainees/FISocketChat/blob/81cff8f2137cf7bda6fc66439fd44ad53c8e74f7/src/app/emotes.json](https://github.com/FI18-Trainees/FISocketChat/blob/81cff8f2137cf7bda6fc66439fd44ad53c8e74f7/src/app/emotes.json)

Comment (Monkmitrad) [reply src/app/emotes.json]: I dislike the whole format too, but will approve never the less.

File: src/app/emotes.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "menuDisplayCode": "<img src='/public/img/peepoWide.png' style='height:35px; width:auto;' alt='peepoWide' title='peepoWide'>"
   },
   "Hackerman": {
-    "replace": "<a target=\"_blank\" href=\"https://www.youtube.com/watch?v=K7Hn1rPQouU\"><img src='/public/img/hackerman.jpg' style='height:100px; width:auto;' alt='Hackerman' title='Hackerman'></a>",
+    "replace": "<a target=\"_blank\" href=\"https://www.youtube.com/watch?v=K7Hn1rPQouU\"<img src='/public/img/hackerman.jpg' style='height:100px; width:auto;' alt='Hackerman' title='Hackerman'></a>",
```
Comment (zaanposni) [inline src/app/emotes.json]: is this on purpose? looks broken to me?

**Comment by SFFan123** [comment]
(╯°□°）╯︵ ┻━┻

**Pull request merged**